### PR TITLE
feat(templates): adding new templates, removing calls to create templates

### PR DESCRIPTION
**Pull request by russorat**
Closes https://github.com/influxdata/influxdb-templates/issues/9
Closes https://github.com/influxdata/influxdb/issues/14505

Describe your proposed changes here.

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/onboarding/actions/index.ts [hunk 1]
```diff
@@ -89,16 +88,6 @@ export const setupAdmin = (params: ISetupParams) => async (
       throw new Error(resp.data.message)
     }
 
-    await client.templates.create({...defaultTemplates.systemTemplate(), orgID})
```
Comment (chnn) [inline ui/src/onboarding/actions/index.ts]: :stars:

File: ui/src/templates/constants/defaultTemplates.ts [hunk 2]
```diff
@@ -1,41 +1,40 @@
...
-  'getting-started': gettingStarted,
-  'local-metrics': localMetrics,
-  System: system,
+export const staticTemplates: {[k: string]: DashboardTemplate} = {
+  Apache: apache,
   Docker: docker,
-  Redis: redis,
-  Nginx: nginx,
+  'getting-started': gettingStarted,
+  Github: github,
```
Comment (chnn) [inline ui/src/templates/constants/defaultTemplates.ts]:
```suggestion
  GitHub: github,
```
if this key is user facing

Comment (ebb-tide) [reply ui/src/templates/constants/defaultTemplates.ts]: it's not, but I checked template names and corrected capitalization there.

**Comment by ebb-tide** [comment]
I went back and forth on whether templateIDs should be more human-readable or influxdbID format- and decided that we might want a way to auto-generate IDs at some point, so wanted to be compatible with that posssibility.
context: 
https://github.com/influxdata/influxdb/issues/14082
https://github.com/influxdata/influxdb/issues/14100
https://github.com/influxdata/influxdb/issues/14101
https://github.com/influxdata/influxdb/issues/14102

**Comment by chnn** [comment]
Ok cool, that makes a lot more sense now. Thanks for the background.

**Pull request merged**